sass/components/_tutorial.scss

```diff
@@ -340,12 +341,12 @@ $tip-index: 1000;
         width: 9px;
         height: 9px;
         margin: 0 5px;
-        background: $black;
+        background: var(--center-channel-color);
 
         &.active {
             @include opacity(1);
 
-            background: $primary-color;
```

Comment:
Nit: remove the unused variables

Reply:
same comment as above ... we currently do not know what impact that would have